packages/modeling/src/primitives/ellipse.js

```diff
@@ -20,24 +24,49 @@ const ellipse = (options) => {
   const defaults = {
     center: [0, 0],
     radius: [1, 1],
+    startAngle: 0,
+    endAngle: (Math.PI * 2),
     segments: 32
   }
-  const { center, radius, segments } = Object.assign({}, defaults, options)
+  let { center, radius, startAngle, endAngle, segments } = Object.assign({}, defaults, options)
 
   if (!isNumberArray(center, 2)) throw new Error('center must be an array of X and Y values')
   if (!isNumberArray(radius, 2)) throw new Error('radius must be an array of X and Y values')
   if (!radius.every((n) => n > 0)) throw new Error('radius values must be greater than zero')
+  if (!isGTE(startAngle, 0)) throw new Error('startAngle must be positive')
+  if (!isGTE(endAngle, 0)) throw new Error('endAngle must be positive')
```

Comment:
is it worth validating that startAngle is less than endAngle?  What happens if they're not?